documentation/docs/install-pmm/install-pmm-client/connect-database/mongodb.md

```diff
@@ -59,7 +59,6 @@ Run the example codes below in a `mongo` session to:
             pwd: "pmm",
             roles: [
                 { role: "explainRole", db: "admin" },
```

Comment:
to not have this duplication maybe it's better to order fields, so `db` on left side and `role` on right side